doc/manual/api/interface.rst_

```diff
@@ -38,118 +44,130 @@ Re2c does not (and cannot) check the definitions, so if anything is missing or d
     A generic API primitive with no arguments.
-    The meaning of ``YYRESTORE`` is to restore the current input position to the
+    ``YYRESTORE`` should restore the current input position to the
     value saved by ``YYBACKUP``.
     The definition should be either function-like or free-form depending on the
     API style (see ``re2c:api:style``).
 
 ``YYBACKUPCTX``
     A generic API primitive with zero arguments.
-    The meaning of ``YYBACKUPCTX`` is to save the current input position as the
-    position of the trailing context, which is later restored by ``YYRESTORECTX``.
+    ``YYBACKUPCTX`` should save the current input position as the
+    position of the trailing context, which is later restored by
+    ``YYRESTORECTX``.
     The definition should be either function-like or free-form depending on the
     API style (see ``re2c:api:style``).
 
 ``YYRESTORECTX``
     A generic API primitive with no arguments.
-    The meaning of ``YYRESTORECTX`` is to restore the trailing context position
+    ``YYRESTORECTX`` should restore the trailing context position
     saved with ``YYBACKUPCTX``.
     The definition should be either function-like or free-form depending on the
     API style (see ``re2c:api:style``).
 
 ``YYRESTORETAG``
     A generic API primitive with one argument ``tag``.
-    The meaning of ``YYRESTORETAG`` is to restore the trailing context position
+    ``YYRESTORETAG`` should restore the trailing context position
     to the value of ``tag``.
     The definition should be either function-like or free-form depending on the
     API style (see ``re2c:api:style``).
 
 ``YYSTAGP``
     A generic API primitive with one argument ``tag``.
-    The meaning of ``YYSTAGP`` is to set ``tag`` value to the current input
+    (``tag`` should represent a pointer or an offset, see the section
+    on Submatch Extraction for details.)
+    ``YYSTAGP`` should set the ``tag`` value to the current input
     position.
     The definition should be either function-like or free-form depending on the
     API style (see ``re2c:api:style``).
 
 ``YYSTAGN``
     A generic API primitive with one argument ``tag``.
-    The meaning of ``YYSTAGN`` is to set ``tag`` value to null (or some default
-    value). The definition should be either function-like or free-form depending
-    on the API style (see ``re2c:api:style``).
+    (``tag`` should represent a pointer or an offset, see the section
+    on  Submatch Extraction for details.)
+    ``YYSTAGN`` should to set the ``tag`` value to null (or
+    some default value).
+    The definition should be either function-like or free-form depending on the
+    API style (see ``re2c:api:style``).
 
 ``YYMTAGP``
     A generic API primitive with one argument ``tag``.
-    The meaning of ``YYMTAGP`` is to append the current position to the history
-    of ``tag``.
+    ``YYMTAGP`` should append the current position to the history
+    of ``tag``. (See the section on Submatch Extraction for details.)
```

Comment:
Can you  make "submatch extraction" lowercase? It should be plain text, not a section name (in case the section name is changed). Sorry if that wasn't clear enough.